src/Sentry.Unity.Editor.iOS/SentryXcodeProject.cs

```diff
@@ -37,8 +58,13 @@ public SentryXcodeProject(string projectRoot) : this(projectRoot, new NativeMain
             INativeOptions sentryNativeOptions)
         {
             _projectRoot = projectRoot;
-            _projectPath = PBXProject.GetPBXProjectPath(projectRoot);
-            _project = new PBXProject();
+
+            var xcodeAssembly = Assembly.Load("UnityEditor.iOS.Extensions.Xcode");
+            var pbxProjectClass = xcodeAssembly.GetType("UnityEditor.iOS.Xcode.PBXProject");
```

Comment:
Best practice question: Should I hold that as a member instead of having to call `_project.GetType()` all the time?

Reply:
Yes, that'd be best

Reply:
should this stuff be done in a static constructor?